kotlin/reakt-native-toolkit/build.gradle.kts

```diff
@@ -60,17 +52,20 @@ kotlin {
     wasmJs {
         nodejs()
     }
+
+    applyDefaultHierarchyTemplate()
+
     sourceSets {
-        commonMain.configure {
+        commonMain {
             dependencies {
                 implementation(libs.kotlinx.coroutines.core)
                 implementation(libs.kotlinx.serialization.json)
             }
         }
 
-        androidMain.configure {
+        androidMain {
             dependencies {
-                implementation("com.facebook.react:react-native:[0.69.0,)") // from node_modules
+                implementation("com.facebook.react:react-android:[0.74.0,)")
```

Comment:
we should document that we now depend on the published react native version, and provide instruction how to exclude this dependencies for older react native version compatibility

Reply:
See https://github.com/voize-gmbh/reakt-native-toolkit/pull/53/commits/f8b2b25b9a08d58d9e3e44f08bc22d080f72f90b#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R40-R41